pypeit/core/telluric.py

```diff
@@ -39,7 +39,7 @@
 # TODO These codes should probably be in a separate qso_pca module. Also pickle functionality needs to be removed.
 # The prior is not used (that was the reason for pickling), so the components could be stored in fits format.
 # npca is not actually required here.
-def init_pca(filename,wave_grid,redshift, npca):
+def init_qso_pca(filename,wave_grid,redshift, npca):
```

Comment:
For consistency with the other methods, should the prefix start with `qso_`, i.e. `qso_init_pca()`

Reply:
That's probably true, changed.